scripts/verify_tests_on_master.bat

```diff
@@ -18,7 +34,7 @@ CD ..\..\..
 REM Now run the tests a bunch of times to try to find flakes (tests that sometimes pass
 REM even though they should be failing).
 @ECHO.
-CALL dart flutter\dev\customer_testing\run_tests.dart --repeat=15 --skip-template registry/*.test || GOTO :END
+CALL dart flutter\dev\customer_testing\run_tests.dart --repeat=15 --skip-template --shards %SHARDS% --shard-index %SHARD_INDEX% --verbose registry/*.test || GOTO :END
```

Comment:
Is the addition of `--verbose` on purpose? Or left-over from debugging?

Reply:
This was on purpose to include more information on the presubmits for this repo. Do you think it should be removed?

Reply:
Seems ok to have it in verbose mode. If it gets too noisy we can dial it down again.